subsys/caf/modules/power_manager.c

```diff
@@ -153,6 +151,18 @@ static void system_off_on_error(void)
 	sys_poweroff();
 }
 
+static void system_off_handler(struct k_work *work)
+{
+	LOG_WRN("System turned off");
+	LOG_PANIC();
```

Comment:
won't panic halt?

Reply:
LOG_PANIC means only to process all logs.